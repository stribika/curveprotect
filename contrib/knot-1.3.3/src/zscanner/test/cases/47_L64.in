$ORIGIN	.
$TTL	1

; OK
@	L64	0	0000:0000:0000:0000	; The simplest case
@	L64	65535	FFFF:FFFF:FFFF:FFFF	; The maximal case
@	L64	0	abcd:ef00:0000:0000	; Lower-case
@	L64	\# 10 00000000000000000000	; Hexadecimal rdata
@	TYPE106	\# 10 00000000000000000000	; TYPE + Hexadecimal rdata
@	TYPE106	0	0000:0000:0000:0000	; TYPE
@	l64	0	0000:0000:0000:0000	; Type in lower-case

; KO
@	L64
@	L64					; Empty rdata
@	L64	\# 0				; Hex empty rdata
@	L64	\#				; Missing hex length
@	L64	65536				; Too big preference
@	L64	0	0000:0000:0000		; Missing label
@	L64	0	0000:0000:0000:0000:0000	; Too many labels
@	L64	0	0000:0000:0000:000	; Missing hex character
@	L64	0	0000:0000:0000-0000	; Bad separator
@	L64	0	0000:0000:0000:x000	; Bad hex character
@	L64	0	0000:0000:0000:0000 x	; Unexpected item
@	L64	\# 11 00000000000000000000 00	; Too long rdata
@	L64	\# 11 00000000000000000000	; Bad rdata length
