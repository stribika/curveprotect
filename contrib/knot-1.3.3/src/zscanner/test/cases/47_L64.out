OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006A
RDATA=00000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006A
RDATA=FFFFFFFFFFFFFFFFFFFF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006A
RDATA=0000ABCDEF0000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006A
RDATA=00000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006A
RDATA=00000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006A
RDATA=00000000000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006A
RDATA=00000000000000000000
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_EBAD_L64_LENGTH
------
WARNG=ZSCANNER_EBAD_L64_LENGTH
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_CHAR_COLON
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_REST
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
